engine/evarutil.mli

```diff
@@ -10,6 +10,7 @@ open Names
 open Term
 open Evd
 open Environ
+open EConstr
```

Comment:
This is to reduce the quantity of changes for the merge. I agree that it can be puzzling by only looking at the mli, but the compiler does a fair job at explaining you why types differ.